[2023-03-01 12:59:00,271] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-25T19:05:00+00:00 [queued]>
[2023-03-01 12:59:00,537] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-25T19:05:00+00:00 [queued]>
[2023-03-01 12:59:00,542] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:59:00,545] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 12:59:00,551] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:59:00,861] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-11-25T19:05:00+00:00
[2023-03-01 12:59:00,934] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-11-25T19:05:00+00:00', '--job-id', '3974', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmpp9_30h1_', '--error-file', '/tmp/tmp1vrvrwit']
[2023-03-01 12:59:00,947] {standard_task_runner.py:77} INFO - Job 3974: Subtask get_name
[2023-03-01 12:59:00,908] {standard_task_runner.py:52} INFO - Started process 16731 to run task
[2023-03-01 12:59:01,559] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-11-25T19:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 12:59:02,167] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-11-25T19:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-25T19:05:00+00:00
[2023-03-01 12:59:02,307] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 12:59:02,393] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221125T190500, start_date=20230301T125900, end_date=20230301T125902
[2023-03-01 12:59:02,913] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 12:59:02,991] {local_task_job.py:146} INFO - Task exited with return code 0
